filter_releases.py

```diff
@@ -58,39 +48,39 @@ def filter_releases(releases, state, label, start_date, end_date, datetype):
       if release.state == state or state == 0:
         if label:
           if label in release.labels:
-              filtered.append(release)
+            filtered.append(release)
         else:
           filtered.append(release)
 
   return filtered
 
 
-def sort(releases, sort_method):
+class Sorting(object):
+  BY_NAME = 1
+  BY_CREATION = 2
+  BY_LAST_MODIFIED = 3
+  BY_LAST_ACTIVE = 4
+
+
+def sort(releases, sort_method, descending):
```

Comment:
Don't we just rely on the DB to sort and filter? What is this class for?

Reply:
This is mainly here for the file adapter, though it is possible that we will have to use this later to sort/filter the releases for certain cases. This is because, as I explained yesterday, we have to assemble some release data from the tasks, and so we most likely won't be able to push all the filtering to SQL

Reply:
I think we probably should make those information available in xcom to facilitate sorting/filtering using sql directly. wdyt?

Reply:
That could certainly make things easier on the ui backend, at least for filtering. We could do something like this:
1. store the hard-to-get release info in xcom
2. query these vars using dag_id & execution_date
3. take the var values and use them to query certain releases

However, I don't think this approach will work for sorting, as there's currently nothing in the SQL database (that I'm aware of) which will allow for sorting of results from a table based on variables not contained in said table